app/client/packages/design-system/widgets/src/components/Sidebar/src/use-mobile.tsx

```
@@ -0,0 +1,27 @@
+
+import { SIDEBAR_CONSTANTS } from "./constants";
+
+export function useIsMobile() {
+  const [isMobile, setIsMobile] = useState<boolean | undefined>(undefined);
+
+  useEffect(() => {
+    const root = document.body.querySelector(
+      "[data-theme-provider]",
+    ) as HTMLButtonElement;
+
+    if (!Boolean(root)) return;
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)